packages/neo-one-cli/src/interactive/createCRUD.js

```diff
@@ -162,6 +180,17 @@ const createResource = ({
     .action(async args => {
       cancel$ = new ReplaySubject();
 
+      if (crud.name === 'delete') {
```

Comment:
What happens if the crud name is overridden to something else? Better check `crud instanceof DeleteCRUD` instead.